.clang-tidy

```diff
@@ -113,3 +113,5 @@ CheckOptions:
     value:           UPPER_CASE
   - key:             readability-identifier-naming.MacroDefinitionPrefix
     value:           MIGRAPHX_
+  - key:             readability-identifier-length.MinimumVariableNameLength
```

Comment:
I think it would be better to disable this check instead.

Reply:
Spec says to disable set to a 0 or a 1.

Reply:
To disable a check you a put `-` in front of the name in the list [here](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/CMakeLists.txt#L149).

Reply:
Done